### PR TITLE
feat(components/atom/videoPlayer): Add autoplay prop to atom videoPlayer

### PR DESCRIPTION
**Pull request by davidmartin84**
### Description, Motivation and Context
Implementing video-related features in Adevinta.

Add a prop to programmatically indicate if the videoplayer should autoplay or not.

### Types of changes

- [x] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: components/atom/videoPlayer/src/index.js [hunk 1]
```diff
@@ -5,21 +5,25 @@ import PropTypes from 'prop-types'
...
+        <Component {...props} />
+      </div>
+    )
+  }
+)
 
 AtomVideoPlayer.displayName = 'AtomVideoPlayer'
 
 AtomVideoPlayer.propTypes = {
+  autoPlay: PropTypes.bool,
```
Comment (jordevo) [inline components/atom/videoPlayer/src/index.js:26]: same as previously commented to @oegea , wouldn't it be best if we stick to boolean-ish names for boolean props? 🤔 

  ```suggestion
  hasAutoPlay: PropTypes.bool,
```

File: components/atom/videoPlayer/src/index.js [hunk 2]
```diff
@@ -5,21 +5,25 @@ import PropTypes from 'prop-types'
...
-    src
-  })
-  return (
-    <div ref={forwardedRef} className={BASE_CLASS}>
-      <Component {...props} />
-    </div>
-  )
-})
+const AtomVideoPlayer = forwardRef(
+  ({autoPlay = false, controls = true, src}, forwardedRef) => {
```
Comment (jordevo) [inline components/atom/videoPlayer/src/index.js:9]: good to see, from a user POV, it's `false` by default

File: components/atom/videoPlayer/src/index.js (hunk 1 above)
Comment (hpintos) [reply components/atom/videoPlayer/src/index.js:26]: IMHO, make sense to leave that property name as is the same that video tag natively uses.

**Review by hpintos — APPROVED** [review]

**Review by quinwacca — APPROVED** [review]

File: components/atom/videoPlayer/src/index.js (hunk 1 above)
Comment (oegea) [reply components/atom/videoPlayer/src/index.js:26]: Yep, we'll review all property names once we have the component in a more advanced status. 

We're not really sure which is the best approach, but currently we are trying to respect the standard html attribute naming.

Maybe if we start to add custom props that does not match with any standard html attribute (which will probably happen), then we'll have to think about changing the approach and name all props without trying to match them with html standard attributes.

So... long story short, we have this on the radar 🤓

**Pull request merged**